stdlib/LinearAlgebra/src/symmetric.jl

```diff
@@ -254,11 +254,19 @@ end
     end
 end
 
+Base._reverse(A::Symmetric, dims::Integer) = (reverse((Matrix(A)); dims))
+Base._reverse(A::Symmetric, dims::Colon) = Symmetric(reverse(Matrix(A)))
+Base._reverse!(A::Symmetric, dims::Colon) = (reverse!(A.data); A)
```

Comment:
```suggestion
```
This method is wrong — you can't reverse in-place because the `uplo` is immutable.